gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/worn/by_Chicken_Delivery_Man/necklaces/necklaces.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
I know those `by_someone` folder are already there but we should probably not use them, sprite should get credit through commit history becasue someone might come and edit them later and then the name of the folder they're in will only hod part of the information anyway.